src/modules/LanItems.rb

```diff
@@ -284,11 +284,6 @@ def GetCurrentName
     #
     # @param itemId [Integer] a key for {#Items}
     def GetDeviceType(itemId)
-      # in such case ifcfg is not existing and /sys might not contain
-      # any device info (especially for virtual devices like vlan)
-      # @type variable is already initialized by @see HardwareDialog
-      # resp its storage handler @see storeHW
-      return @type if @operation == :add
```

Comment:
Removing / moving this line is safe enough?, there is no some corner case / leftover when adding a new interface which still uses GetDeviceType? If that is the case then it LGTM

Reply:
yes. the line was moved closer to place where it is needed to free "generic" method